### PR TITLE
KEEPALIVE/RC-MLX5: added processing of non-empty arbiter

### PR DESCRIPTION
**Pull request by hoopoepg**
- optimization: added evaluation of non-empty arbiter. in case
  if EP is scheduled in arbiter then keepalive action
  is not needed

### CODE REVIEW

File: src/uct/ib/rc/accel/rc_mlx5_ep.c [hunk 1]
```diff
@@ -541,8 +541,15 @@ uct_rc_mlx5_ep_check(uct_ep_h tl_ep, unsigned flags, uct_completion_t *comp)
     uint64_t dummy = 0;
 
     UCT_CHECK_PARAM(comp == NULL, "Unsupported completion on ep_check");
-    UCT_RC_CHECK_TX_CQ_RES(&iface->super, &ep->super);
 
+    if (ucs_arbiter_group_is_scheduled(&ep->super.arb_group)) {
```
Comment (brminich) [inline src/uct/ib/rc/accel/rc_mlx5_ep.c:545]: why is this needed? If some pending exists, we are missing TX resources and therefore the check on line 552 will return `NO_RESOURCE` . Seems duplicate

Comment (hoopoepg) [reply src/uct/ib/rc/accel/rc_mlx5_ep.c:545]: not duplicate: in case if there is anything in arbiter then we assume that arbiter will provide any operation instead of keepalive and UCS_OK is returned without any additional action

Comment (brminich) [reply src/uct/ib/rc/accel/rc_mlx5_ep.c:545]: pending op can do flush(no op), send on  another ep, or do not do any send at all.

**Comment by brminich** [comment]
@hoopoepg, can we close it now?
